scss/_variables.scss

```diff
@@ -44,7 +44,7 @@ $popup-padding-x: 0 !default;
 $popup-padding-y: 0 !default;
 
 $header-padding-x: $padding-x !default;
-$header-padding-y: $padding-y !default;
+$header-padding-y: $padding-x !default;
```

Comment:
That particular bit was discussed with @yordanov -- the padding is equal on all sides and is equal to to `$padding-x` i.e. changing the latter must and will change the padding in a predictable way.